### PR TITLE
Merge remote-tracking branch 'origin/master' into feature/test_rw_analyzer.py_

### PR DESCRIPTION
**Pull request by arialfie759**
fixed documentation and did unit testing for rw_analyzer.py

### CODE REVIEW

**Comment by chantisnake** [comment]
couple of over-indenting error, if you can fix that.

File: LICENSE [hunk 1]
```diff
@@ -1,3 +1,5 @@
+
```
Comment (chantisnake) [inline LICENSE]: revert this

File: README.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
...
+Lexos is a suite of tools designed to facilitate the computational analysis of literary and historical texts. It offers an integrated workflow in which the pre-processing ("scrubbing"), analysis, and visualization steps can be accomplished in a single, web-based environment. Scrubbing features include handling punctuation, stop words, markup tags, and character consolidations, as well as document segmentation, culling, and n-gram tokenization. Analytical tools include basic document statistics, hierarchical and k-means cluster analysis, cosine similarity ranking, and z-score analysis. Visualizations include word and bubble clouds, comparative "multiclouds" (which can be used to analyze data from [MALLET](http://mallet.cs.umass.edu/)-produced topic models) and rolling window analysis. Analytical tools produce line, PCA, Voronoi cell, and dendrogram graphs. Each of the tools has export functionality.
```
Comment (chantisnake) [inline README.md]: probably useful change, but not on this branch

File: README.md [hunk 3]
```diff
@@ -44,7 +44,7 @@ Lexos works on Chrome and Firefox. Other browsers are not supported, and some fe
 
 ## License information
 See the file LICENSE for information on the
-terms & conditions for usage and a DISCLAIMER OF ALL WARRANTIES.
+Terms & Conditions for usage and a DISCLAIMER OF ALL WARRANTIES.
```
Comment (chantisnake) [inline README.md]: probably useful change, but not on this branch

File: lexos/static/package-lock.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (chantisnake) [inline lexos/static/package-lock.json]: deletes this file

File: README.md (hunk 3 above)
Comment (arialfie759) [inline README.md]: Should I make a new branch for these changes?

**Comment by chantisnake** [comment]
Your test doesn't seem to cover a_word_line function. Can you implement that?

**Comment by arialfie759** [comment]
a_word_line was the function that was written incorrectly so I thought you didn't want me to test it yet?

**Comment by chantisnake** [comment]
That makes sense, Thank you.

Your pull request will be merged soon.

**Pull request merged**